code/modules/vore/fluffstuff/custom_boxes_vr.dm

```diff
@@ -140,3 +140,16 @@
 		new /obj/item/weapon/card/id/centcom/fluff/aro(src)
 		new /obj/item/weapon/reagent_containers/hypospray/fluff/aronai(src)
 		new /obj/item/clothing/glasses/sunglasses/omnihud(src)
+
+//ivymoomoo:Ivy Baladeva
+/obj/item/weapon/storage/backpack/messenger/sec/fluff/ivymoomoo
+	name = "Ivy's Courier"
+	desc = "A bag resembling something used by college students. Contains items for ''MooMoo''."
+
+	New()
+		..()
+		new /obj/item/clothing/head/beretg(src)
+		new /obj/item/weapon/card/id/centcom/fluff/ivyholoid(src)
```

Comment:
This line needs to be  changed to
new /obj/item/weapon/card/id/fluff/ivyholoid(src)
so it'll compile correctly & work correctly.